jscomp/others/js_bigint.res

```diff
@@ -1,3 +1,74 @@
 /*** JavaScript BigInt API */
 
-type t
+@val
+/**
+Parses the given `string` into a `bigint` using JavaScript semantics. Return the
+number as a `bigint` if successfully parsed, `null`, `undefined`, `_NaN` otherwise.
+
+## Examples
+
+```rescript
+/* returns 123n */
+Js.Bigint.fromString("123")
+
+/* returns 0n */
+Js.Bigint.fromString("")
+
+/* returns 17n */
+Js.Bigint.fromString("0x11")
+
+/* returns 3n */
+Js.Bigint.fromString("0b11")
+
+/* returns 9n */
+Js.Bigint.fromString("0o11")
+```
+*/
+external fromStringExn: string => bigint = "BigInt"
```

Comment:
What's the exception? It would be good to have an example to see how to catch it.

Reply:
JS BigInt constructor throws error such as `Uncaught SyntaxError` in case of passing a non-numeric value. I'll add an example .